apps/govquests-frontend/src/app/quests/page.tsx

```diff
@@ -0,0 +1,10 @@
+"use server";
+
+import api from "@/lib/api";
+import QuestsList from "./(components)/QuestsList";
+
+export default async function Quests() {
+  const quests = await api("quests");
+
+  return <QuestsList quests={quests} />;
```

Comment:
`#would` be cool to leverage this to implement a suspense boundary with a loading state